code/modules/mob/living/basic/space_fauna/paper_wizard/paper_wizard.dm

```diff
@@ -0,0 +1,196 @@
+		/datum/ai_planning_subtree/find_paper_and_write,
+	)
+
+/datum/ai_planning_subtree/attack_obstacle_in_path/paper_wizard
+	target_key = BB_FOUND_PAPER
+	attack_behaviour = /datum/ai_behavior/attack_obstructions/paper_wizard
+
+/datum/ai_behavior/attack_obstructions/paper_wizard
+	action_cooldown = 0.4 SECONDS
+	can_attack_turfs = TRUE
+	can_attack_dense_objects = TRUE
+
+/datum/ai_planning_subtree/targeted_mob_ability/wizard_mimic
+	ability_key = BB_WIZARD_MIMICS
+	finish_planning = FALSE
+
+/datum/ai_planning_subtree/use_mob_ability/wizard_summon_minions
+	ability_key = BB_WIZARD_SUMMON_MINIONS
+	finish_planning = FALSE
+
+/datum/ai_behavior/find_and_set/empty_paper
+	action_cooldown = 10 SECONDS
+
+/datum/ai_behavior/find_and_set/empty_paper/search_tactic(datum/ai_controller/controller, locate_path, search_range)
+	var/list/empty_papers = list()
+
+	for(var/obj/item/paper/target_paper in oview(search_range, controller.pawn))
+		if(target_paper.is_empty())
+			empty_papers += target_paper
+
+	if(empty_papers.len)
+		return pick(empty_papers)
+
+/mob/living/basic/paper_wizard/copy
+	desc = "'Tis a ruse!"
+	health = 1
+	maxHealth = 1
+	alpha = 200
+	faction = list(FACTION_STICKMAN)
+	melee_damage_lower = 1
+	melee_damage_upper = 5
+
+	ai_controller = /datum/ai_controller/basic_controller/wizard_copy
+	///the master that summoned us
+	var/mob/living/basic/paper_wizard/original
+
+/mob/living/basic/paper_wizard/copy/grant_abilities()
+	return
+
+/mob/living/basic/paper_wizard/copy/grant_loot()
+	return
+
+//Hit a fake? eat pain!
+/mob/living/basic/paper_wizard/copy/death(gibbed)
+	. = ..()
+
+	for(var/mob/living/damaged in oview(5, src))
+		if(faction_check_mob(damaged, exact_match = FALSE))
+			continue
+		damaged.adjustBruteLoss(50)
+
+
+/mob/living/basic/paper_wizard/copy/examine(mob/user)
+	. = ..()
+	if(isobserver(user))
+		. += span_notice("It's an illusion - what is it hiding?")
+	else
+		qdel(src) //I see through your ruse!
+
+/datum/ai_controller/basic_controller/wizard_copy
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic,
+	)
+
+	ai_movement = /datum/ai_movement/basic_avoidance
+	idle_behavior = /datum/idle_behavior/idle_random_walk
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/simple_find_target,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree,
+	)
+
+//fancy effects
+/obj/effect/temp_visual/paper_scatter
+	name = "scattering paper"
+	desc = "Pieces of paper scattering to the wind."
+	layer = ABOVE_NORMAL_TURF_LAYER
+	plane = GAME_PLANE
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "paper_scatter"
+	anchored = TRUE
+	duration = 5
+	randomdir = FALSE
+
+/obj/effect/temp_visual/paperwiz_dying
+	name = "craft portal"
+	desc = "A wormhole sucking the wizard into the void. Neat."
+	layer = ABOVE_NORMAL_TURF_LAYER
+	plane = GAME_PLANE
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "paperwiz_poof"
+	anchored = TRUE
+	duration = 18
```

Comment:
```suggestion
	duration = 1.8 SECONDS
```
Sometimes I think it does actually make sense to put the duration in deciseconds when you're matching the frames of an icon animation but it's the convention not to